src/main/java/eu/bbmri_eric/negotiator/database/model/DiscoveryServiceSynchronizationJob.java

```diff
@@ -0,0 +1,45 @@
+package eu.bbmri_eric.negotiator.database.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import java.time.LocalDateTime;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.hibernate.annotations.UuidGenerator;
+
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Builder
+@Entity(name = "DiscoveryServiceSynchronizationJob")
```

Comment:
is this necessary?

Reply:
Yes, this is the Entity carrying on the job statistiscs (to be improved in the second part of the PR)